run.sh

```diff
@@ -106,6 +106,8 @@ service cups restart
 echo "export EYEOS_UNIX_USER=$EYEOS_UNIX_USER" > /tmp/global.env
 echo "export BUS_ADDRESS_HOST=$BUS_ADDRESS_HOST" >> /tmp/global.env
 echo "export BUS_SUBSCRIPTION=$BUS_SUBSCRIPTION" >> /tmp/global.env
+echo "export EYEOS_BUS_MASTER_PASSWD=$EYEOS_BUS_MASTER_PASSWD" >> /tmp/global.env
```

Comment:
Just to make damn sure, this master password is in reality the user card, isn't it?